driver-core/src/main/com/mongodb/event/CommandSucceededEvent.java

```diff
@@ -36,31 +36,54 @@ public final class CommandSucceededEvent extends CommandEvent {
 
     /**
      * Construct an instance.
+     *
      * @param requestContext the request context
+     * @param operationId the operation id
      * @param requestId the request id
      * @param connectionDescription the connection description
      * @param commandName the command name
      * @param response the command response
      * @param elapsedTimeNanos the non-negative elapsed time in nanoseconds for the operation to complete
-     * @since 4.4
+     * @since 4.10
      */
-    public CommandSucceededEvent(@Nullable final RequestContext requestContext, final int requestId,
+    public CommandSucceededEvent(@Nullable final RequestContext requestContext, final long operationId, final int requestId,
```

Comment:
In what circumstances would end-users need to use these constructors?

Reply:
Generally I would only expect an end user to use these only for testing.  But we haven't distinguished them as _not_ a part of the API, so that means they are part of the API as public members of public classes